zh/backup-restore-snapshot-perf.md

```diff
@@ -107,3 +107,7 @@ EBS 卷快照恢复阶段包含以下阶段，详细信息，参考[基于 EBS 
 | 200 GB   | 500 GB  | 7000IOPS/400 MiB/s | 21 min   |
 | 500 GB   | 1024 GB | 7000IOPS/400 MiB/s | 25 min   |
 | 1024 GB  | 3500 GB | 7000IOPS/400 MiB/s | 34 min   |
+
+> **注意：**
+>
+> 恢复数据性能曲线并不是完全线性单调递增，这里卷数据指的是 TiKV 实际使用的用户数据，而 TiKV 启动和数据恢复阶段实际加载和扫描的卷数据块可能不同，同时受 AWS 卷数据块下载网络速度等的影响，导致恢复数据性能曲线并不完全线性单调递增。
```

Comment:
```suggestion
> 数据恢复性能曲线并非完全线性单调递增，这是因为，卷数据是 TiKV 实际使用的用户数据，而启动 TiKV 和恢复数据时，可能需要加载和扫描不同的卷数据块，另外，AWS 卷数据块下载网络速度等因素对恢复性能也会产生影响。
```

Reply:
fixed in https://github.com/pingcap/docs-tidb-operator/pull/2169/commits/e34683931ef9c4c531dfdc857f45a2597b396a44